### DISCUSSION TITLE
[Feature Request] Publish a Flatpak version

### DISCUSSION THREAD

**Discussion by regunakyle** [vial-kb/vial-gui#174] [Q&A] [answered]
I have just started using VIAL and it is awesome. VIA cannot recognize for my Keychron K8 Pro for some reason, but VIAL manage to save the day.

Publishing a Flatpak has some advantages for users, e.g. updating via console (`flatpak upgrade`) or app store of GNOME/KDE, sandbox support, etc.. A nice to have feature IMO.

**Comment by xyzz** [comment, marked as answer]
Please use https://feedback.vial.today/ for feature requests
